src/convertToObject.js

```diff
@@ -1,12 +1,18 @@
 'use strict';
 
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = sourceString
+    .split(';')
+    .map((element) => element.split(':'))
+    .map((element) => element.map((str) => str.trim()))
+    .filter((element) => element.length > 1)
+    .reduce((res, [key, value]) => {
+      res[key] = value;
+
+      return res;
+    }, {});
+
+  return result;
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable.

Good example:
const result = sourceString
  .split(';')
  .map(element => element.split(':'))
  .map(element => element.map(str => str.trim()))
  .filter(element => element.length > 1)
  .reduce((res, [key, value]) => {
    res[key] = value;
    return res;
  }, {});